elements/pfe-jump-links/pfe-jump-links-nav.ts

```diff
@@ -33,16 +34,31 @@ function isQueryable(x: Node): x is Document|ShadowRoot {
   return x instanceof Document || x instanceof ShadowRoot;
 }
 
+export class NavStuckEvent extends ComposedEvent {
+  constructor(
+    public stuck: Boolean,
+  ) {
+    super('stuck');
+  }
+}
+
+export class ActiveItemEvent extends ComposedEvent {
+  constructor(
+    index: keyof Array,
+  ) {
+    super('active-item');
```

Comment:
WDYT about this name? I was thinking `change` might be better, but we already use `change` in panel...

Reply:
maybe:
`active-item` => `change`
`change` => `content-change`

I think that whatever name we decide for panel contents changing we'll want to standardize for other elements reacting to mutation observer changes.

Reply:
let's defer that discussion to later, k?

Reply:
Sounds good.  I made the changes to the event names and updated the tests to cover the new events.